src/test/java/teammates/test/cases/logic/EmailGeneratorTest.java

```diff
@@ -203,14 +203,31 @@ public void testGenerateFeedbackSessionEmails() throws IOException {
         assertTrue(hasStudent1ReceivedEmail);
         assertTrue(hasInstructor1ReceivedEmail);
         
+        ______TS("send summary of all feedback sessions of course email");
+        
+        EmailWrapper email = new EmailGenerator().generateFeedbackSessionResendAllLinksEmail(
+                session.getCourseId(), student1);
+
+        subject = String.format(EmailType.STUDENT_EMAIL_CHANGED.getSubject(), course.getName());
+        
+        hasStudent1ReceivedEmail = false;
+        hasInstructor1ReceivedEmail = false;
```

Comment:
No need to set this to false if it's not being used; if other tests are added later which use the same variable name, it can be set to false there instead.